evaluations/evaluation_aragog.py

```diff
@@ -136,7 +143,7 @@ def parameter_tuning(questions, answers, out_path: str):
                 print(f"Running evaluation")
                 results, inputs = run_evaluation(questions, answers, retrieved_contexts, predicted_answers, embedding_model)
                 eval_results = EvaluationRunResult(run_name=name_params, inputs=inputs, results=results)
-                eval_results.score_report().to_csv(f"{out_path}/score_report_{name_params}.csv", index=False)
+                eval_results.score_report().to_csv(f"{out_path}/score_report_{name_params}.csv")
```

Comment:
I don't what to change this because it will affect the notebook that reads the CSV files, and I would rather leave it as it is.

Reply:
I am testing this with the results notebook and you need this to have the metric names as column names.